zk/stages/stage_sequence_execute_utils.go

```diff
@@ -238,7 +238,8 @@ func prepareL1AndInfoTreeRelatedStuff(sdb *stageDb, decodedBlock *zktx.DecodedBa
 		}
 	}
 
-	if l1TreeUpdate != nil {
+	// we only want GER and l1 block hash for indexes above 0 - 0 is a special case
+	if l1TreeUpdate != nil && l1TreeUpdateIndex > 0 {
```

Comment:
question about this one and blockNum == 1 -- that doesn't affect here, right?

Reply:
yep, we're clear here as there is another code path to handle the injected batch.  In there we build a fake l1 update with the GER and L1 block hash from the event on the L1